listenbrainz/webserver/static/js/info.js

```diff
@@ -1,31 +1,42 @@
-document.getElementById("show-hide-token").addEventListener("click", function() {
-    var token = document.getElementById("auth-token");
-    var btn = document.getElementById("show-hide-token");
+document
+  .getElementById("show-hide-token")
+  .addEventListener("click", function () {
+    const token = document.getElementById("auth-token");
+    const btn = document.getElementById("show-hide-token");
 
-    if (token.type === 'password') {
-        token.type = 'text';
-        btn.className = 'btn btn-info glyphicon glyphicon-eye-close';
-        btn.title = "Hide user token";
+    if (token.type === "password") {
+      token.type = "text";
+      btn.className = "btn btn-info glyphicon glyphicon-eye-close";
+      btn.title = "Hide user token";
     } else {
-        token.type = 'password';
-        btn.className = 'btn btn-info glyphicon glyphicon-eye-open';
-        btn.title = "Show user token";
+      token.type = "password";
+      btn.className = "btn btn-info glyphicon glyphicon-eye-open";
+      btn.title = "Show user token";
     }
-});
+  });
 
 function copy(token) {
-    token.select();
-    document.execCommand("copy");
+  token.select();
+  document.execCommand("copy");
 }
 
-document.getElementById("copy-token").addEventListener("click", function() {
-    var token = document.getElementById("auth-token");
-    if (token.type === 'password') {
-        token.type = 'text';
-        copy(token);
-        token.type = 'password';
-    }
+document.getElementById("copy-token").addEventListener("click", function () {
+  const token = document.getElementById("auth-token");
+  if (token.type === "password") {
+    token.type = "text";
     copy(token);
-    var copyButton = document.getElementById("copy-token");
-    copyButton.textContent = "Copied!"
+    token.type = "password";
+  }
+  copy(token);
+  const copyButton = document.getElementById("copy-token");
+  copyButton.textContent = "Copied!";
+});
+
+const caller = document.getElementById("delete-account");
+caller.addEventListener("mouseenter", function () {
+  const randX = Math.floor(Math.random() * (window.innerWidth - 100));
+  const randY = Math.floor(Math.random() * (window.innerHeight - 100));
+  console.log([randX, randY]);
+  caller.style.left = `${randX}px`;
+  caller.style.top = `${randY}px`;
```

Comment:
So… my suggestion was a joke, not a real suggestion :p
I don't think we should actually implement it. As a user I would find it very annoying (and probably still funny)

Reply:
Done